cypress/e2e/shared/explorer.test.ts

```diff
@@ -769,6 +772,16 @@ describe('DataExplorer', () => {
             cy.getByTestID('raw-data-table').should('exist')
             cy.getByTestID('raw-data--toggle').click()
             cy.getByTestID('giraffe-axes').should('exist')
+
+            cy.reload()
+
+            cy.wait(2000)
+
+            cy.getByTestID('selector-list defbuck').should('be.visible')
+
+            cy.getByTestID('selector-list m').should('be.visible')
+
+            cy.getByTestID('selector-list v').should('be.visible')
```

Comment:
this condition kind of captures it, but add the "is visible _AND_ selected" condition to each of these